packages/lage/src/logger/initReporters.ts

```diff
@@ -31,6 +32,9 @@ export function initReporters(config: Config) {
     reporters.push(new AdoReporter());
   }
 
+  // Will always include CustomReporter as well to pass metadata.
+  reporters.push(new CustomReporter());
```

Comment:
I think the right thing to do here is to add an additional arg like `--reporter lage-custom-reporter.js`

Reply:
The idea here maybe to have the fallback be a catchall - if the arg is a file that exists, import it and use it.

Reply:
The reason that I implemented it this way is because, if we go via the args route then any consuming repo who wants to use this custom reporter will have to change all their calls to lage to add this new argument. Whereas in current way, it will automatically be available for everyone without any changes, anyone who wants to use will just have to add the file, otherwise it will be no op. Thought?

Reply:
@kenotron gentle reminder

Reply:
This is not how lage tool is expected to be configured. We add arguments for things we need to change or change the behavior via the one lage.config.js (which we relent as an implicit JS tooling standard way of configuration). Please add the arg to get this merged!

Reply:
Added